docs/store-operations/shipping/msf-international-enhancements.mdx

```diff
@@ -0,0 +1,75 @@
+# Shipping - International Enhancements for Multi-Storefront
+
+When you sell internationally or to mulitple customer segments, you may want the available shipping methods for a shipping zone to depend on the shopper's region or segment.
+
+To localize your storefront, you can have zones that display different methods depending on the storefront channel. 
+Specify the storefront channel(s) where you want a shipping method to appear when you create a method for a zone. Shoppers only see the shipping method for the zone if it applies to a channel.
+
+## Limitations
+
+You can't assign the following to a specific channel:
+- The free shipping method on [static shipping quotes](https://support.bigcommerce.com/s/article/Shipping-Methods#static).
+- [Pickup methods](/docs/integrations/buy-online-pick-up-in-store#pickup-methods-api) on stores using Buy Online Pickup in Store.
+
+We don't support multiple accounts of the same carrier (e.g., using distinct accounts for UPS)
```

Comment:
`don't` or `do not`.
I know in formal letters it is best to avoid contractions when possible, but I am not sure if this is considered formal or not.

Reply:
We have contractions in some areas in our docs (not a standard yet)